applications/tari_console_wallet/src/ui/state/app_state.rs

```diff
@@ -450,13 +468,18 @@ impl AppStateInner {
         wallet: WalletSqlite,
         base_node_selected: Peer,
         base_node_config: PeerConfig,
+        balance_enquiry_cooldown_period: u64,
```

Comment:
This should be a Duration type
```suggestion
        balance_enquiry_cooldown_period: Duration,
```